FastAPI changes for CoE LLM Routing Kit

snova-kwasia opened the pull request.

snova-petrojm commented on CoE_jump_start/config.yaml:
```diff
@@ -1,10 +1,19 @@
+    "sambaverse_model_name": "Meta/Meta-Llama-3-8B-Instruct"
+    "coe": True #set as true if using Sambastudio CoE endpoint
+    "select_expert": "Meta-Llama-3.1-70B-Instruct" #set if using FastCoE, Sambaverse or SambaStudio CoE llm expert
+    #fastApi CoE expert name -> "llama3-8b"
```
this kit is not compatible with fastapi, i think we should remove this line

snova-kwasia replied:
Now removed thanks @snova-petrojm

snova-petrojm commented:
Changes to config.yaml and coe_jump_start.py lgtm.

However, can't open the Streamlit app, getting this error below. Can you please review the requirements file?
ImportError: cannot import name 'pre_init' from 'langchain_core.utils'

Also, some sections of the README need updates. For example:
Starter kit usage -> Using the Python script:
Expert Mode and 4. Expert Mode are same (i think you're missing explanation for E2E with vector db)

snova-luiss commented:
Changes to README:

- Configuration: API and LLM Settings needs to be updated like config.yaml file
- .env file variables names examples should be updated as well
- Python command instructions should be: python src/use_CoE_model.py expert --query "What is the capital of France?"

snova-kwasia commented:
@snova-petrojm @snova-luiss 

Thanks for the review. Have addressed your comments in the latest commit as well as tested the notebook. Please could you re-review then we can get this merged in. Thanks
